File: NopTHAndroid/Cau3_BanThan/app/src/main/java/ntu/thu_62132086/cau4_banthan/MainActivity.java
package ntu.thu_62132086.cau4_banthan;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Khởi tạo TextViews
        TextView textViewName = findViewById(R.id.textViewName);
        TextView textViewDepartment = findViewById(R.id.textViewDepartment);
        TextView textViewHobbies = findViewById(R.id.textViewHobbies);

        // Thiết lập thông tin cá nhân
        textViewName.setText("Tên: Nguyễn Ngọc Anh Thư");
        textViewDepartment.setText("Khoa: Công Nghệ Thông Tin");
        textViewHobbies.setText("Sở thích: Hát");
    }
}

File: NopTHAndroid/BottomNavigationRecylerView/app/src/main/java/ntu/thu_62132086/bottomnavigation_recylerview/MainActivity.kt
package ntu.thu_62132086.bottomnavigation_recylerview
import android.R
import android.R.id
import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.google.android.material.bottomnavigation.BottomNavigationView


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activityMain)
        val bottomNav = findViewById<BottomNavigationView>(id.bottom_navigation)
        bottomNav.setOnNavigationItemSelectedListener(navListener)

        // as soon as the application opens the first
        // fragment should be shown to the user
        // in this case it is algorithm fragment
        val commit = supportFragmentManager.beginTransaction()
            .replace(id.fragment_container, AlgorithmFragment()).commit()
    }

    private val navListener =
        BottomNavigationView.OnNavigationItemSelectedListener { item: MenuItem ->
            // By using switch we can easily get
            // the selected fragment
            // by using there id.
            var selectedFragment: Fragment? = null
            val itemId = item.itemId
            if (itemId == id.gorithm) {
                selectedFragment = AlgorithmFragment()
            } else if (itemId == id.course) {
                selectedFragment = CourseFragment()
            } else if (itemId == id.profile) {
                selectedFragment = ProfileFragment()
            }
            // It will help to replace the
            // one fragment to other.
            if (selectedFragment != null) {
                supportFragmentManager.beginTransaction()
                    .replace(id.fragment_container, selectedFragment).commit()
            }
            true
        }
}